docs/format/source/format_release_notes.rst

```diff
@@ -12,6 +12,7 @@ Minor changes
 - Deprecated `ImagingRetinotopy` neurodata type. (#565)
 - Modified `OptogeneticSeries` to allow 2D data, primarily in extensions of `OptogeneticSeries`. (#564)
 - Added optional ``stimulus_template`` column to ``IntracellularStimuliTable`` as part of the ``IntracellularRecordingsTable``. (#545)
+- Added optional `bounds` attribute to `SpatialSeries.data`. (#567)
```

Comment:
```suggestion
- Added optional ``bounds`` attribute to ``SpatialSeries.data``. (#567)
```

RST format uses double backtick for code. Otherwise it will be italicized. I messed this up above too.